pallets/capacity/src/lib.rs

```diff
@@ -212,6 +249,36 @@ pub mod pallet {
 	pub type UnstakeUnlocks<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, UnlockChunkList<T>>;
 
+	/// stores how many times an account has retargeted, and when it last retargeted.
```

Comment:
I think some documentation for UnstakeUnlocks would help clarify the purpose of that storage.